pkg/cmd/pr/status/status_test.go

```diff
@@ -76,7 +76,10 @@ func runCommand(rt http.RoundTripper, branch string, isTTY bool, cli string) (*t
 }
 
 func initFakeHTTP() *httpmock.Registry {
-	return &httpmock.Registry{}
+	registry := &httpmock.Registry{}
+	// SPIKE: perhaps we should inject a mock detector into the command instead?
```

Comment:
We inject mock detectors for tests around `template` and `issue closing`. We could follow the same pattern here. However, if we split `detector.PullRequestFeatures` as described in another comment, then we can probably get away with making no changes at all, since the test already elides the introspection request for the new types.

Reply:
I don't have strong feelings here but I don't generally like hiding assertions inside helper functions away from other assertions. We almost always call `registry.Verify()` in our tests which turns this into an assertion.

Reply:
Understood and agree. An unsaid part of the comment was "I put this here because I wanted to quickly validate this worked instead of copying it into all the tests".